docs/guide/advanced/api.md

```diff
@@ -0,0 +1,139 @@
+# API
+
+YOURLS comes with an API wrapper, `yourls-api.php` at the root of your install.
+The API allows to:
+
+- Generate or get existing short URLs, with sequential or custom keyword.
+- Get some statistics about your links: top clicked links, least clicked links, newest links.
+- Return various output format: JSON, XML, or simple raw text.
+- Authenticate either with login/password or using a secure [passwordless](passwordless-api) mechanism.
+- Even define your own custom API requests and actions.
+
+## Usage
+
+Parameters has to be sent to `http://your-own-domain-here.com/yourls-api.php` either via GET or POST (remember to URL-encode parameters if via GET).
```

Comment:
```suggestion
Parameters must be sent to `http://your-own-domain-here.com/yourls-api.php` either via GET or POST. (Remember to URL-encode parameters if via GET!)
```

Does the website support "admonitions" like GH's extended Markdown? The parenthetical could be an "important" aside, like below.

----

Parameters must be sent to `http://your-own-domain-here.com/yourls-api.php` either via GET or POST.

> [!IMPORTANT]
> Remember to URL-encode parameters if using the GET request method.